packages/styles/combobox.css

```diff
@@ -111,8 +111,28 @@
   position: relative;
 }
 
+.Combobox .TextFieldWrapper {
+  flex-wrap: wrap;
+}
+
 .Combobox .TextFieldWrapper input[type='text'] {
   margin-right: 1.25rem;
+  flex-grow: 1;
+  width: unset;
+  min-width: var(--input-min-width);
```

Comment:
Why was `min-width` added here? This creates an accessibility issue at lower viewport widths.